<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestForm</class>
 <widget class="QWidget" name="TestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px solid rgb(255,255,255);background-color:rgb(0,0,0);</string>
  </property>
  <widget class="QLabel" name="widget_MonitorWin1">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>27</y>
     <width>470</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="widget_MonitorWin2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>27</y>
     <width>470</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="widget_MonitorWin4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>470</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_CarChose" native="true">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>533</y>
     <width>470</width>
     <height>115</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_Car1">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Car2">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Car3">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Car4">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Car5">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Car6">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Car7">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Car8">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>50</y>
      <width>45</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarChose">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="widget_MonitorWin3">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>280</y>
     <width>470</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
